### PR TITLE
Changed active from primitive boolean to Boolean

### PR DESCRIPTION
**Pull request by gmackay**

### CODE REVIEW

File: vcm/src/main/java/uk/gov/cshr/vcm/controller/VacancyController.java [hunk 1]
```diff
@@ -47,6 +45,10 @@
     @ApiOperation(value = "Create a Vacancy", nickname = "create")
     public ResponseEntity<Vacancy> create(@RequestBody Vacancy vacancy) {
 
+        if ( vacancy.getActive() == null ) {
```
Comment (seamussingh) [inline vcm/src/main/java/uk/gov/cshr/vcm/controller/VacancyController.java]: This should probably go in the private createVacancy method which is also called from the save method for safety sake.

Comment (reachsaurabh) [reply vcm/src/main/java/uk/gov/cshr/vcm/controller/VacancyController.java]: hey guys , this is the last PR , can this be actioned if there is risk of a potential bug

Comment (reachsaurabh) [reply vcm/src/main/java/uk/gov/cshr/vcm/controller/VacancyController.java]: if its not a breaking change , leave this for now , merging it

Comment (gmackay) [reply vcm/src/main/java/uk/gov/cshr/vcm/controller/VacancyController.java]: That's no problem.  To be honest, it should really go in a service method which overrides the default but I though that was overkill for such a small issue.

**Pull request merged**